File: plusmall-operator-web/src/main/webapp/js/service/contentCatService.js
app.service('contentCatService',function ($http) {

    //查找
    this.search=function (pageNum, pageSize) {
        return $http.get('../contentCat/search.do?pageNum='+pageNum+'&pageSize='+pageSize);
    }

    //删除
    this.delete=function (ids) {
        return $http.get('../contentCat/delete.do?ids='+ids);
    }

    //添加
    this.add=function (contentCat) {
        return $http.post('../contentCat/add.do',contentCat);
    }

    //根据id查找单个广告分类
    this.findOne=function (id) {
        return $http.get('../contentCat/findOne.do?id='+id);
    }

    //更新
    this.update=function (contentCat) {
        return $http.post('../contentCat/update.do?',contentCat);
    }
})

File: plusmall-operator-interface/src/main/java/com/plusmall/operator/ContentCatService.java
package com.plusmall.operator;

import com.plusmall.commons.PageResult;
import com.plusmall.model.TbContentCategory;

import java.util.List;

/**
 * @Description:
 */
public interface ContentCatService {
	public PageResult search(int pageNum,int pageSize);
	public void delete(Long[] ids) throws NullPointerException;
	public void add(TbContentCategory contentCategory) throws NullPointerException;
	public TbContentCategory findOne(Long id);
	public void update(TbContentCategory contentCategory) throws NullPointerException;
	public List<TbContentCategory> findAll();
}
